packages/engine/src/gltf/GLTFLoaderFunctions.ts

```diff
@@ -783,6 +786,20 @@ const loadMaterial = async (options: GLTFParserOptions, materialIndex: number) =
 
   await Promise.all(promises)
 
+  //apply deltas
+  const deltaState = getState(SceneDeltaState)
+  const sourceDelta = deltaState[options.documentID.replaceAll(/\?hash=[^-]+/g, '')]
```

Comment:
let's move this regex to GLTFComponent and add a helper function so we can reuse it, rather than redefining it everywhere time we need it